src/api/api.js

```diff
@@ -0,0 +1,8 @@
+import AxiosInstance from '../auth/axiosInstance';
+
+const CREATE_TEAM = '/api/v1/protected/teams';
+export const createTeam = async team => AxiosInstance.post(CREATE_TEAM, team);
```

Comment:
The big downside to this pattern is that you probably want to try catch anything thats async

Reply:
Doesn't the .then.catch accomplish the same thing? Pretty sure 
```
try { 
  // code1
} catch (e) {
  // code2
}
```
is equivalent to
```
.then((response) => code1).catch((error) => code2)
```

Reply:
Yep, the .then / .catch pattern is equivalent as long as we actually conform to that pattern in the calling code.

Reply:
So which do we want? I prefer this by a ton haha